src/xrpld/net/detail/RPCCall.cpp

```diff
@@ -1606,7 +1606,10 @@ fromNetwork(
     constexpr auto RPC_REPLY_MAX_BYTES = megabytes(256);
 
     using namespace std::chrono_literals;
-    auto constexpr RPC_NOTIFY = 10min;
+    // Wietse: used to be 10m, but which backend
+    // ever requires 10 minutes to respond?
+    // Lower = prevent stacking pending calls
```

Comment:
I think this comment is more of a PR comment than something that needs to be in the code